lib/matplotlib/backend_bases.py

```diff
@@ -1464,7 +1464,7 @@ def on_pick(event):
             line = event.artist
             xdata, ydata = line.get_data()
             ind = event.ind
-            print('on pick line:', np.array([xdata[ind], ydata[ind]]).T)
+            print('on pick line:', xdata[ind], ydata[ind])
```

Comment:
I bet this array call was here to get reasonably truncated repr:

```suggestion
            print(f'on pick line: {ind}: ({xdata[ind]:.3f}, {ydata[ind]:.3f})')
```

Reply:
fair enough, fixed.